src/app/repository/platform-repository.service.ts

```diff
@@ -68,6 +68,10 @@ export class PlatformRepositoryService {
         return this.ipc.request("fetchPlatformData");
     }
 
+    checkForPlatformUpdates(): Observable<any> {
```

Comment:
This method doesn't belong here, it has nothing to do with managing the active platform user data.